config/checkstyle/checkstyle.xml

```diff
@@ -119,6 +119,10 @@
         <module name="SuperClone" />
         <module name="MutableException" />
         <module name="AvoidStarImport" />
+        <module name="ImportControl">
+            <property name="file" value="${config_loc}/import-control.xml"/>
+            <property name="path" value="cloud-aws/src/main/java.*$"/>
```

Comment:
what do this?

Reply:
It is used to check that no service uses aws api without wrapping it in a client class - so that the error message mapping will be applied to every call